Buildings/Experimental/OpenBuildingControl/CDL/Sources/package.mo

```diff
@@ -0,0 +1,13 @@
+within Buildings.Experimental.OpenBuildingControl.CDL;
+package Sources "Package with models for source signals that are used with controllers"
+  extends Modelica.Icons.SourcesPackage;
+
+annotation (
+preferredView="info", Documentation(info="<html>
+This package contains components models for sources
+that are used with controllers.
+For additional models, see also
+<a href=\"modelica://Modelica.Blocks.Sources\">
+Modelica.Blocks.Sources</a>.
```

Comment:
Please remove `For additional models....` as we only want to use `CDL` models when expressing sequences.